Tasks/Common/VstsAzureHelpers_/Utility.ps1

```diff
@@ -225,6 +225,9 @@ function ConvertTo-Pfx {
     }
 
     $openSSLExePath = "$PSScriptRoot\openssl\openssl.exe"
+    $env:OPENSSL_CONF = "$PSScriptRoot\openssl\openssl.cnf"
```

Comment:
Is this fix required only for PS based task?

Reply:
yes. there is no issue in node based tasks